CMakeLists.txt

```diff
@@ -522,6 +522,12 @@ if (x86ish)
   set(RR_SOURCES ${RR_SOURCES} src/test/x86/cpuid_loop.S)
 endif()
 
+option(asan "Build with address santitizer enabled.")
+if (asan)
+  set(ASAN_FLAGS "-fno-omit-frame-pointer -fsanitize=address")
```

Comment:
Is `-fno-omit-frame-pointer` necessary here? If this was because of less-than-useful stacktraces from leaks, there's `ASAN_OPTIONS=fast_unwind_on_malloc=false`. I'm wondering if `-fno-omit-frame-pointer` might be causing some of the test timeout issues?

Reply:
> Is `-fno-omit-frame-pointer` necessary here?

I am just using it because the usage example I looked at had it too. Have to test if that makes a difference.
At least even with this the stack e.g. in 7214d9614 was not complete.